apps/backoffice-v2/src/domains/workflows/hooks/queries/useWorkflowDefinitionQuery/useWorkflowDefinitionQuery.ts

```diff
@@ -0,0 +1,14 @@
+import { useQuery } from '@tanstack/react-query';
+import { workflowsQueryKeys } from '../../../query-keys';
+
+export const useWorkflowDefinitionQuery = ({
```

Comment:
Keep consistency with other queries.
```suggestion
export const useWorkflowDefinitionByIdQuery = ({
```